src/precice/SolverInterface.hpp

```diff
@@ -4,6 +4,7 @@
 #include <set>
 #include <string>
 #include <vector>
+#include "precice/Version.h"
```

Comment:
Do we want to include it by default? Your changelog entry sounds more like the user has to include the file additionally.

Reply:
Including it by default means that you don't need to include the header in the adapter.
This makes your adapter forward compatible as the header doesn't exist in current releases.